README.tfdoc.hcl

```diff
@@ -0,0 +1,342 @@
+header {
+  image = "https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg"
+  url   = "https://mineiros.io/?ref=terraform-module-template"
+
+  badge "build" {
+    image = "https://github.com/mineiros-io/terraform-module-template/workflows/Tests/badge.svg"
```

Comment:
please run `make template/adjust` to replace all references to template repo with current repo name